core/services/chainlink/testdata/config-full.toml

```diff
@@ -7,6 +7,7 @@ ShutdownGracePeriod = '10s'
 FeedsManager = true
 LogPoller = true
 UICSAKeys = true
+BlockchainPlugins = false
```

Comment:
@jmank88 - I presume that we want this to always be false unless overridden, until we have tests for plugins - that sound correct?